pkg/vault/contracts/lib/HooksConfigLib.sol

```diff
@@ -35,7 +35,7 @@ library HooksConfigLib {
     using WordCodec for bytes32;
     using HooksConfigLib for PoolConfigBits;
 
-    // #region Bit offsets for hooks config
+    // Bit offsets for hooks config
```

Comment:
```suggestion
    // Bit offsets for hooks config.
```

Reply:
Here and elsewhere - it was like this before, but non-inline comments should end in a period.